File: Python/Ex.4.py
v1 = [0,0,0,0]
v2 = [0,0,0,0]
v3 = [0,0,0,0]

for v in range(0,4):
  v1[v] = int(input("Insira um número: "))
  v2[v] = int(input("Insira um número: "))

  v3[v] = v1[v] + v2[v]
  print(v3[v])

File: JavaScript/Ex.4.js
let v1 = [4]
let v2 = [4]

for(let n = 0; n < 4; n++){
  v1[n] = Number(prompt("Insira um número: "))
  v2[n] = Number(prompt("Insira um número: "))

  let v3 = []
  v3 = v1[n] + v2[n]
  console.log(v3)
} 